lambda_src/drivers/process_https.py

```diff
@@ -43,17 +42,13 @@ def process_row(
         raise ValueError('URL scheme must be HTTPS.')
 
     req_host = u.hostname
-    req_kwargs = parse_header_dict(kwargs)
-    req_headers = {
-        k: v.format(**req_kwargs)
-        for k, v in (
-            loads(headers)
-            if headers.startswith('{')
-            else parse_header_dict(headers)
-            if headers
-            else {}
-        ).items()
-    }
+    req_headers = (
+        loads(headers)
+        if headers.startswith('{')
+        else parse_header_dict(headers)
```

Comment:
err, I think as long as we call `parse_header_dict`, we will be decrypting things without checking what they are. could we also remove the `decrypt_if_encrypted` call from there?

there's a second (properly gated) call on line 58, which should be the only one left.

Reply:
oo right, makes sense. removed it.